CHANGELOG.md

```diff
@@ -2,6 +2,31 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [0.18.0](https://github.com/ERC725Alliance/erc725.js/compare/v0.17.2...v0.18.0) (2023-08-03)
+
+### ⚠ BREAKING CHANGES
+
+- new encoding for static value types (not arrays `[]`) (#288)
+- change `ArrayLength` value from `uint256` to `uint128` (#287)
+
+### Features
+
+- add encoding / decoding for `bytes[CompactBytesArray]` ([#261](https://github.com/ERC725Alliance/erc725.js/issues/261)) ([8d3e4e9](https://github.com/ERC725Alliance/erc725.js/commit/8d3e4e994957dbf1f54eef4572b072bc308c61bc))
```

Comment:
This was part of the previous release

<img width="1211" alt="image" src="https://github.com/ERC725Alliance/erc725.js/assets/31145285/4b2b315f-6b11-464d-879e-e9c6884e490d">

Reply:
good catch - we need to fix that release ci